src/libraries/System.Private.CoreLib/src/System/Threading/PortableThreadPool.cs

```diff
@@ -43,43 +45,55 @@ internal sealed partial class PortableThreadPool
         private int _cpuUtilization; // SOS's ThreadPool command depends on this name
         private short _minThreads;
         private short _maxThreads;
-        private readonly LowLevelLock _maxMinThreadLock = new LowLevelLock();
 
         [StructLayout(LayoutKind.Explicit, Size = Internal.PaddingHelpers.CACHE_LINE_SIZE * 6)]
```

Comment:
Like in the beginning of the struct it was supposed to have an extra cache line at the end to avoid overlapping the last fields with fields of other objects. I moved `gateThreadRunningState` into the same cache line as the previous field because they are used together in the hot paths and that field doesn't change frequently. Now the size leaves an extra cache line at the end as expected.